index.html

```diff
@@ -4,9 +4,17 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/index.css">
     <title>Photo Gallery</title>
 </head>
 <body>
-    
+    <header id="header">
```

Comment:
This is fine but I would also consider the slightly cleaner approach with the header such as: 

```<header id="header"></header>``` all on one line.